src/backoff.ts

```diff
@@ -19,7 +19,7 @@ export async function backoff(
   maxJitterMs = 1000,
   rng: () => number = Math.random,
 ): Promise<void> {
-  return await new Promise((resolve) =>
```

Comment:
please verify if the removal of this return do not change the behavior of the feature. I also had the same issue on this PR https://github.com/Lichtblick-Suite/electron-socket/pull/1/files#diff-0683ac1ac56f263656c251be476cdcdece1f1a2b8753f41870adfd6d4ae83bffL106 and @luluiz helped me on that